enterprise/server/remote_execution/containers/docker/docker.go

```diff
@@ -445,6 +449,10 @@ func copyOutputs(reader io.Reader, result *interfaces.CommandResult, stdio *inte
 	if *commandutil.DebugStreamCommandOutputs {
 		stdout, stderr = io.MultiWriter(stdout, os.Stdout), io.MultiWriter(stderr, os.Stderr)
 	}
+	if !stdio.DisableOutputLimits {
+		stdout = commandutil.LimitStdOutErrWriter(stdout)
+		stderr = commandutil.LimitStdOutErrWriter(stderr)
+	}
```

Comment:
If we added + used a "commandutil.LimitedBuffer" instead of `bytes.Buffer` above, then we wouldn't need this special `DisableOutputLimits` option, right? There are two cases: either we are buffering the outputs in memory, in which case we should be using a limited buffer, or stdout/stderr are pipes, in which case the persistentworker package is responsible for limiting the output.